src/schedule-and-details/requirements-section/index.jsx

```diff
@@ -129,7 +129,7 @@ RequirementsSection.propTypes = {
   entranceExamMinimumScorePct: PropTypes.string,
   isPrerequisiteCoursesEnabled: PropTypes.bool.isRequired,
   onChange: PropTypes.func.isRequired,
-  isEditable: PropTypes.bool,
+  isEditable: PropTypes.bool.isRequired,
```

Comment:
You can remove `isEditable` from the `defaultProps` section if it is now required.